pkg/omf/cli/omf.install.fish

```diff
@@ -36,7 +36,7 @@ function omf.install -a type_flag name_or_url
   if test -e $OMF_PATH/db/$parent_path/$name_or_url
     set target $parent_path/$name_or_url
   else
-    set -l local_name (basename $name_or_url)
+    set -l local_name (basename $name_or_url | sed "s/pkg-//;s/plugin-//;s/theme-//")
```

Comment:
1. Anchor your regex, yo. 
2. This is a regex. You could totally do it with `(theme|pkg|plugin)` instead of three replacements.
3. Number 2 also keeps you from transforming my new `pkg-theme-plugin-maker` package folder into just `maker` :)